packages/server/test/integration/cypress_spec.js

```diff
@@ -1731,26 +1702,6 @@ describe('lib/cypress', () => {
       })
     })
 
-    // NOTE: skipped because we want to ensure this is captured in v10
```

Comment:
This workflow is covered by a system test `system-tests/test/network_error_handling_spec.js`

Reply:
We also have it here for launchpad:

https://github.com/cypress-io/cypress/blob/develop/packages/launchpad/cypress/e2e/config-files-error-handling.cy.ts#L242-L271